src/Cookie/CookieJar.php

```diff
@@ -98,6 +98,25 @@ public static function shouldPersist(
         return false;
     }
 
+    /**
+     * Finds and returns the cookie based on the name
+     *
+     * @param string $name cookie name to search for
+     * @return SetCookie|null cookie that was found or null if not found
+     */
+    public function getCookieByName($name)
+    {
+        $foundCookie = null;
+        foreach($this->cookies as $cookie) {
+            if(strtolower($cookie->getName()) === strtolower($name)) {
```

Comment:
Please add a space between `if` and the opening `(`.

I think `strcasecmp` would be nicer here as well.